src/dopameme/search.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, **kwargs):
         self.modifier = kwargs.get("modifier", rand_mod) or rand_mod
 
     @property
-    def default_str(self):
+    def default_str(self) -> str:
```

Comment:
This is why I'm on the fence about Typing, the clue is in the name, this definitely returns a string! If your function doesn't tell you what it does, ask it again.

Reply:
The point of it for the most part is to enable static analysis of the code in order to catch bugs without having to run a program.

I go both ways with it but I'm mostly in favor after a while of using them.